docs/developer/app-toolkit.md

```diff
@@ -0,0 +1,159 @@
+# TCE App Toolkit Adding Packages How-To
+
+## Summary
+
+This document describes how to add an additional TCE package to the App Toolkit meta-package.
+
+## Assumptions
+
+This document is intended for maintainers of official repositories, as defined in the [community-edition/docs/packaging](https://github.com/vmware-tanzu/community-edition/tree/main/docs/packaging) README. Currently the role is limied to VMware employees.
+
+## Concepts
+
+### Meta-package
+
+A meta-package is defined as a TCE package that installs a set of other already present TCE packages. It is not intended to add new functionality on its own, but deploy a set of existing functionalities to achieve a particular outcome.
+
+### App Toolkit 
+
+App Toolkit is an instance of a meta-package with a goal of allowing TCE users to deploy a set of TCE packages sufficient to deploy a workload. App Toolkit itself provides little/no novel functionality, but comprises an opinion about what TCE packages are required to achieve the outcome. 
+
+The OSS App Package concept is comparable to proprietary Tanzu Application Platform, although reduced in scope.
+
+### Configuration of App Toolkit and Included TCE Packages.
+
+Configuration is deferred to the included TCE package. For instance, if end users are interested in changing the default behavior of, say, knative-serving, they will learn about and provide configuration for knative-serving, not app-toolkit.
+
+### Similarities and Differences to TCE Packaging of Upstream Projects
+
+TCE provides guidance about how to include other upstream projects into TCE - e.g. kpack, knative, etc., as described in the [TCE Documentation](https://tanzucommunityedition.io/docs/latest/package-creation-step-by-step/). Principly it describes how to
```

Comment:
```suggestion
TCE provides guidance about how to include other upstream projects into TCE - e.g. kpack, knative, etc., as described in the [TCE Documentation](https://tanzucommunityedition.io/docs/latest/package-creation-step-by-step/). Principally it describes how to
```

Reply:
Also need to fix up all the lint warnings from the checks - those are too numerous to provide suggestions for.